kotest-assertions/kotest-assertions-core/src/jvmTest/kotlin/com/sksamuel/kotest/matchers/file/FileMatchersTest.kt

```diff
@@ -45,14 +45,13 @@ import io.kotest.matchers.shouldBe
 import io.kotest.matchers.shouldNot
 import io.kotest.matchers.string.shouldEndWith
 import io.kotest.matchers.string.shouldMatch
+import sun.font.FontUtilities.isWindows
```

Comment:
Fix me

Reply:
This might be useful https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/SystemUtils.html